src/kano_safe_mode/check_kano_safe_mode.py

```diff
@@ -97,13 +93,12 @@ def main():
     if check_safe_mode_hotkeys():
         logger.warn("Safe Mode requested")
         set_safe_mode_requested()
-        run_cmd('kano-checked-reboot safeboot systemctl reboot --force')
+        run_cmd('kano-checked-reboot safeboot systemctl reboot')
         return
 
 
 # Decide if needed and move to another file if it is.
-
-def is_safe_mode():
+# def is_safe_mode():
     """TODO"""
 
-    return (os.system('systemctl --quiet is-active kano-safe-mode.target') == 0)
+#    return (os.system('systemctl --quiet is-active kano-safe-mode.target') == 0)
```

Comment:
We don't need this? If not let's just ditch it - we have it in the git history if needed.